config/Run2_2016/samples.yaml

```diff
@@ -15,6 +15,7 @@ GLOBAL:
     - Flag_eeBadScFilter
     #- Flag_hfNoisyHitsFilter
     #- Flag_BadChargedCandidateFilter
+  badMET_flag_runs: []
```

Comment:
here and in other configs: don't need to create empty lists => undo modifications